deeprank/features/Edesolv.py

```diff
@@ -0,0 +1,411 @@
+import warnings
+
+from pdb2sql import pdb2sql, interface
+from deeprank.features import FeatureClass
+from copy import deepcopy
+from io import StringIO
+import numpy as np
+
+try:
+    from Bio.PDB import PDBParser, SASA
+
+except ImportError:
+    warnings.warn('Biopython module not found. Make sure you have Biopython 1.79 or later')
+    raise
+
+
+class Edesolv(FeatureClass):
+
+    # init the class
+    def __init__(self, pdb_data):
+        """Compute the energy of desolvation feature.
+
+        Biopython 1.79 or later is required for this feature.
+
+        Args:
+            pdb_data (list(byte) or str): pdb data
+
+        Example:
+            >>> edesolv = Edesolv('1AK4.pdb')
+            >>> edesolv.get_feature()
+        """
+        self.pdb = pdb_data
+        print('PDB DATA: ')
+        print(pdb_data)
+        print('TYPE: ' + str(type(pdb_data)))
```

Comment:
Remove the print or use a logger if printing is necessary

Reply:
It is not useful anymore. Will be removed in the next commit to this branch.